replica/server.go

```diff
@@ -141,10 +141,11 @@ func (s *Server) Create(size int64) error {
 	state, _ := s.Status()
 
 	if state != Initial {
-		fmt.Println("STATE = ", state)
+		if err := s.Open(); err != nil {
```

Comment:
need to do complete thorough of the callers of this function..
can you provide the list of callers and scenarios where it gets triggered and the impact with the changes?

Reply:
above comment applies to all the functions which changed the way they are supposed to work

Reply:
we should consider removing Create also from REST APIs list..